src/components/Carousel/_carousel.scss

```diff
@@ -79,17 +79,31 @@
   &-dot__container {
     align-items: center;
     display: flex;
-    height: 8px;
+    height: 10px;
     justify-content: center;
-    min-width: 8px;
+    margin: 2px;
+    min-width: 10px;
     padding: 1px;
     scroll-snap-align: center;
-    width: 8px;
+    width: 10px;
```

Comment:
there's two spots in `CarouselDots` that need to be adjusted for the new width

https://github.com/PhillipsAuctionHouse/seldon/blob/8c3760bce1849eb5606c4c2480a915e0f7ab0ac0/src/components/Carousel/CarouselDots.tsx#L48-L49

8 needs to be replaced with 10 (probably should just set this to a variable), and 10 needs to be replaced with 11. What this does is center the dot that has just been clicked to

and

https://github.com/PhillipsAuctionHouse/seldon/blob/8c3760bce1849eb5606c4c2480a915e0f7ab0ac0/src/components/Carousel/CarouselDots.tsx#L110-L111

replace 8 with 10